build in ubuntu 32bit fail

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':core:runtime'.
> Could not find opencv-linux-x86_32.jar (org.bytedeco.javacpp-presets:opencv:3.0.0-1.1).
  Searched in the following locations:
      https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/opencv/3.0.0-1.1/opencv-3.0.0-1.1-linux-x86_32.jar
```

The real url of 32bits jar doesn't contain "_32":

```
https://repo1.maven.org/maven2/org/bytedeco/javacpp-presets/opencv/3.0.0-1.1/opencv-3.0.0-1.1-linux-x86.jar
```

--- a/build.gradle
+++ b/build.gradle
@@ -76,7 +76,7 @@ allprojects {
     }
 }
 
-def os = osdetector.classifier.replace("osx", "macosx").replace("windows-x86_32", "windows-x86")
+def os = osdetector.classifier.replace("osx", "macosx").replace("windows-x86_32", "windows-x86").replace("x86_32", "x86")
 def arch = osdetector.arch.replace("x86_64", "x64")
 
 project(":core") {

--- a/build.gradle
+++ b/build.gradle
@@ -76,7 +76,7 @@ allprojects {
     }
 }
 
-def os = osdetector.classifier.replace("osx", "macosx").replace("windows-x86_32", "windows-x86").replace("x86_32", "x86")
+def os = osdetector.classifier.replace("osx", "macosx").replace("x86_32", "x86")
 def arch = osdetector.arch.replace("x86_64", "x64")
 
 project(":core") {

